.circleci/config.yml

```diff
@@ -149,10 +149,10 @@ commands:
             sudo chmod +x /usr/local/go/bin/kustomize1
             kustomize1 version
       - run:
-          name: Install Kustomize v2.0.3
+          name: Install Kustomize v2.1.0
           command: |
             set -x
-            [ -e /tmp/dl/kustomize ] || curl -sLf -C - -o /tmp/dl/kustomize https://github.com/kubernetes-sigs/kustomize/releases/download/v2.0.3/kustomize_2.0.3_linux_amd64
```

Comment:
ok, this needs a few more changes, because it won't actually overwrite the existing version, 

* on line 155 change `/tmp/dl/kustomize` to `/tmp/dl/kustomize2.1.0`
* on line 156 change `/tmp/dl/kustomize` to `/tmp/dl/kustomize2.1.0`

check to see if build passes

* on line 88 insert `- dl-v5` and replaces line 169 with `- dl-v5`

Reply:
I did the first two bullet points. Unit tests pass, but the build fails on a step I don't understand the point of. https://circleci.com/gh/argoproj/argo-cd/1395?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

Is that failure a sign of something else bad?

I did not do "on line 88 insert - dl-v5 and replaces line 169 with - dl-v5". I don't understand what that means. I haven't used circleci before. You want that block to look like:
```
      - restore_cache:
          keys:
            - dl-v5
            - dl-v3
```

?

Reply:
It should actually be
```yaml
 - restore_cache:
          keys:
            - dl-v5
            - dl-v4
```

I didn't explain, but CircleCI uses these keys to restore files from the cache. It makes builds faster. By upgrading Kustomize, we change the flies, so we need to invalidate the cache by bumping the version number.

https://circleci.com/docs/2.0/caching/

Reply:
I've just made the change for you.